luda-adventure/src/app/game/menu/render/render_start_menu.rs

```diff
@@ -58,7 +60,13 @@ fn render_start_new_button<'a>(height: Px) -> table::TableCell<'a> {
             BUTTON_FILL_COLOR,
             BUTTON_PADDING,
             [MouseButton::Left],
-            |_| namui::event::send(menu::Event::StartNewButtonClicked),
+            |_| {
+                ImageLoader::request_load();
+                MapLoader::request_laod("first".to_string(), Xy::new(8.tile(), 6.tile()));
+                SaveLoad::request_auto_save_on();
+                Menu::request_close();
+                Menu::request_change_tab(menu::Tab::InGame);
+            },
         )
```

Comment:
It's a new pattern.
Since the number of events increases, It may be bad for performance.
But we can figure out what button actually doing in a second.